oembed/v1.0/oembed.md

```diff
@@ -0,0 +1,64 @@
+# ClickView oEmbed API
+
+[oEmbed](https://oembed.com/) is an open format designed to allow embedding content from a website into another page.
+
+The ClickView oEmbed API can be used to obtain the the necessary information to embed ClickView videos into your website.
+
+
+## Resource URL
+
+```http
+GET https://online.clickview.com.au/oembed
+GET https://online.clickview.co.uk/oembed
+GET https://online.clickview.co.nz/oembed
+```
+
+## Parameters
+
+| Name | Required | Description | Default | Example |
+| ---- | -------- | ----------- | ------- | ------- |
+| url | required | The URL of the resource you are trying to embed || `https://clickv.ie/w/{short-code}` |
+| format | optional | The response format. Supported values are `json` and `xml`. | `json` | `json` |
+
+## Supported URLs
+oEmbed is supported on the following pages:
+
+**Shared Links**:
+* `https://clickv.ie/w/{short-code}`
+
+**Videos**:
+
+* `share?sharecode={share-code}`
+* `share/embed?sharecode={share-code}`
```

Comment:
Only the above two should be in here. Everything below it doesn't and/or shouldn't work properly.